User Guide/Projects.md

```diff
@@ -884,21 +915,23 @@ You can see the newly created topic being added in **Projects** page > click **P
 
 ![Image not Available](/assets/Fig49.png)
 
-You can add more topics within the topic on your own project by selecting from topic folders and clicking the ![Image not Available](/assets/icon9.png) icon.
+You can add more topics within the topic on their own project by selecting from topic folders and clicking the ![Image not Available](/assets/icon9.png) icon.
```

Comment:
"You can add more topics within the topic on their own project by selecting from topic folders" ->  "You can add more topics under a topic by selecting from the topic folder"

Reply:
done